amoro-core/src/main/java/org/apache/amoro/io/reader/CombinedDeleteFilter.java

```diff
@@ -126,11 +126,23 @@ protected CombinedDeleteFilter(
           case EQUALITY_DELETES:
             if (deleteIds.isEmpty()) {
               deleteIds = ImmutableSet.copyOf(ContentFiles.asDeleteFile(delete).equalityFieldIds());
+              firstDeleteFilePath = delete.path().toString();
             } else {
-              Preconditions.checkArgument(
-                  deleteIds.equals(
-                      ImmutableSet.copyOf(ContentFiles.asDeleteFile(delete).equalityFieldIds())),
-                  "Equality delete files have different delete fields");
+              Set<Integer> currentDeleteIds =
+                  ImmutableSet.copyOf(ContentFiles.asDeleteFile(delete).equalityFieldIds());
+              if (!deleteIds.equals(currentDeleteIds)) {
+                LOG.warn(
```

Comment:
Logging a separate warning message does not seem to be a good practice. 
How about adding this information to the exception's error message, then we can find it in the exception stack information and the Dashboard UI.